packages/document/docs/en/plugins/list/plugin-lightningcss.mdx

```diff
@@ -80,6 +80,14 @@ pluginLightningcss({
 });
 ```
 
+- **Example:** Enable `lightningcss-loader` only in the development environment, still use `postcss-loader` in the production environment
```

Comment:
Prefer use `production build` instead of `production environment` in the English doc:

```suggestion
- **Example:** Enable `lightningcss-loader` only in the development build, still use `postcss-loader` in the production build
```

Reply:
done. Is there a knack to this?